### PR TITLE
fix(chore): remove unnecessary methods and prefixes

### PR DESCRIPTION
**Pull request by lubber-de**
## Description
Since 2.9.0 we limited the browser support, so many browser methods are available everywhere. There is no need to check for them beforehand so i removed those.

### CODE REVIEW

File: src/definitions/modules/modal.js [hunk 1]
```diff
@@ -35,13 +35,7 @@
...
             queryArguments = [].slice.call(arguments, 1),
 
-            requestAnimationFrame = window.requestAnimationFrame
-                || window.mozRequestAnimationFrame
-                || window.webkitRequestAnimationFrame
-                || window.msRequestAnimationFrame
-                || function (callback) {
-                    setTimeout(callback, 0);
-                },
+            requestAnimationFrame = window.requestAnimationFrame,
```
Comment (mvorisek) [inline src/definitions/modules/modal.js:38]: can be used directly

Comment (lubber-de) [reply src/definitions/modules/modal.js]: Yes, sure. I removed the declaration entirely now by e4e0f8226aa78e3ae185cebe7c4a09b6f4e223f6

Comment (mvorisek) [reply src/definitions/modules/modal.js]: but access still needs to be done using `window.`, otherwise more eslint undef errors

Comment (lubber-de) [reply src/definitions/modules/modal.js]: but the linter CI does not trigger an error?

Comment (lubber-de) [reply src/definitions/modules/modal.js]: eslintrc.js already got
```
    env: {
        browser: true,
``` so eslint knows about the global functions

Comment (mvorisek) [reply src/definitions/modules/modal.js]: I do not think so, the under rule is disabled.

**Pull request merged**